include/openspace/rendering/renderable.h

```diff
@@ -129,6 +132,10 @@ class Renderable : public properties::PropertyOwner {
     bool _shouldUpdateIfDisabled = false;
     RenderBin _renderBin = RenderBin::Opaque;
 
+    // An optional renderbin that renderables can use for certain components, in cases
+    // where all parts of the renderable should not be rendered in the same bin
+    std::optional<RenderBin>_secondaryRenderBin = std::nullopt;
```

Comment:
The default constructor for `std::optional` initializes it with a `std::nullopt`, so no need for the assignment here

Reply:
Also missing space between `>` and `_`